csharp/src/Ice/Internal/Network.cs

```diff
@@ -803,7 +803,7 @@ public static List<EndPoint> getAddresses(
         return addresses;
     }
 
-    public static IPAddress[] getLocalAddresses(int protocol, bool includeLoopback, bool singleAddressPerInterface)
+    public static IPAddress[] getLocalAddresses(int protocol, bool singleAddressPerInterface)
```

Comment:
My other changes made the includeLoopback parameter always true, so I simplified this code.